w1r3/java/src/main/java/W1R3.java

```diff
@@ -191,7 +191,15 @@ private void worker(
             .setUnit("ns/By{CPU}")
             .build();
 
-    Instrumentation instrumentation = new Instrumentation(latencyHistogram, cpuPerByteHistogram);
+    var allocatedBytesPerByteHistogram =
+        meter
+            .histogramBuilder("ssb/w1r3/memory")
+            .setExplicitBucketBoundariesAdvice(makeMemoryHistogramBoundaries())
+            .setUnit("By")
```

Comment:
The units are `bytes / bytes` or `1{memory}`.

Reply:
fixed